libdevcore/SHA3.h

```diff
@@ -31,26 +31,26 @@ namespace dev
 
 // SHA-3 convenience routines.
```

Comment:
Can you also touch this line?  (It's not a problem in this PR.  I'm just begging.)